src/openrct2/drawing/X8DrawingEngine.cpp

```diff
@@ -406,19 +406,21 @@ void X8DrawingEngine::DrawAllDirtyBlocks()
             uint32_t yy;
             for (yy = y; yy < dirtyBlockRows; yy++)
             {
+                bool shouldBreak = false;
```

Comment:
Rename this variable to `reachedEndOfRow`. (It's important to know _why_ we have to break.)